### DISCUSSION TITLE
Model set up questions

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#61] [Q&A]
As Im going through the model files I have some questions. There is no requirement to answer right away but these keep my thoughts ordered. We can discuss as we go.

**Comment by brianlangseth-NOAA** [comment]
What was the logic for having survey fleets spatially separated as well as combined into a separate coastal fleet? Is the reason that we need a separate coastal fleet because fitting three trends to a single areas is going to be problematic?

**Reply by okenk** [reply]
It seems like using three time series for a coastwide model is a bad idea?

**Comment by brianlangseth-NOAA** [comment]
I see our accumulator age in the .dat file is 40, yet our age bins in the data only go up to age 35. Should we update our age bins to go to 40? Probably not going to make a large effect and the easier choice would be to set the value in .dat file to 35. I also wonder whether at this age (longevity is ~80) we are missing some of the female dynamics, or whether growth is changing, and thus we should set comps and .dat file age higher. 

Manual suggests setting the age bins and lengths bins (for data) so as to be similar even though they are set separately. At minimum lets check to confirm these bins are reasonable.
